system-tests/projects/protocol/cypress/support/commands.ts

```diff
@@ -0,0 +1,36 @@
+// ***********************************************
```

Comment:
can we just do `supportFile: false` in the configuration for this and remove this file?

Reply:
Updated to `supportFile: false`, [73fef85](https://github.com/cypress-io/cypress/pull/27659/commits/73fef8513d89e7d92f15a74d51467f8ad42e086c)